opt: update test output to convert `inverted-lookup` join to `inverted` join

mneverov opened the pull request:
opt: update test output to convert `inverted-lookup` join to `inverted` join

fixes #55765

Release note: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/56180)
<!-- Reviewable:end -->

mneverov committed:
opt: update test output to convert `inverted-lookup` join to `inverted` join

fixes #55765

Release note: none

mneverov commented:
@rytaft could you please take a look?

I changed the `exec/execbuilder/testdata/inverted_index` with `make testoptlogic TESTFLAGS="-rewrite"` but other files I modified manually. Is there a way to rewrite them automatically? If not - does it make sense to add this functionality?

rytaft reviewed (approved):
Hi @mneverov -- thanks for this contribution! <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

In the future, to rewrite the other test files, you can use the command `make test PKG=./pkg/sql/opt/... TESTFLAGS='-rewrite=true'` (some of the sub-packages don't have data-driven tests and will throw errors, but don't worry about that).

bors r+

> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/56180#-:-ML8cnSQ7mFR_uRXxlKo:b4tpwwd)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->
